host/xtest/xtest_1000.c

```diff
@@ -1106,9 +1106,6 @@ static void xtest_tee_test_1013(ADBG_Case_t *c)
 		TEEC_AllocateSharedMemory(&xtest_teec_ctx, &shm)))
 		return;
 
-	Do_ADBG_BeginSubCase(c, "Busy loop with %zu parallel threads",
```

Comment:
This information would be interesting to see, perhaps print it before the loop calling this function.